apps/frontend/public/static/export/xccdfTemplate.xml

```diff
@@ -0,0 +1,111 @@
+<?xml version="1.0" encoding="utf-8"?>
+<?xml-stylesheet type='text/xsl' href='STIG_unclass.xsl'?>
+{{#profiles}}
+<Benchmark xmlns:dc="http://purl.org/dc/elements/1.1/" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xmlns:cpe="http://cpe.mitre.org/language/2.0" xmlns:xhtml="http://www.w3.org/1999/xhtml" xmlns:dsig="http://www.w3.org/2000/09/xmldsig#" xsi:schemaLocation="http://checklists.nist.gov/xccdf/1.1 http://nvd.nist.gov/schema/xccdf-1.1.4.xsd http://cpe.mitre.org/dictionary/2.0 http://cpe.mitre.org/files/cpe-dictionary_2.1.xsd" id="XCCDF_HDF_EXPORT_STIG" xml:lang="en" xmlns="http://checklists.nist.gov/xccdf/1.1">
+   <status date="{{date}}">accepted</status>
+   <title>{{title}}</title>
+   <description>{{description}}</description>
+   <notice id="terms-of-use" xml:lang="en"></notice>
+   <front-matter xml:lang="en"></front-matter>
+   <rear-matter xml:lang="en"></rear-matter>
+   <reference href="https://cyber.mil">
+      <dc:publisher>{{maintainer}}</dc:publisher>
+      <dc:source>{{source}}</dc:source>
+   </reference>
+   <version>{{hdfConvertersVersion}}</version>
```

Comment:
We should have a todo/new issue (if it doesn't already exist) discussing Aaron's ask about providing information on what the original file might have been in those attributes that he'd dug up.